examples/data-sources/jira_field/by_name/data-source.tf

```diff
@@ -0,0 +1,11 @@
+data "jira_field" "example" {
+  name = "Team URL"
+}
+
+output "field_id" {
+  value = data.jira_field.example.field_id
+}
+
+output "field_type" {
+  value = data.jira_field.example.field_type
+}
```

Comment:
Missing newline at end of file. Please add a newline character at the end of this file.